install/mips/include/google/protobuf/any.h

```diff
@@ -0,0 +1,107 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Same here.
Why we shall add Protocol buffers one more time to the repo because of MIPS?
We shall use existing structure for that and just customize build steps to make it work.

Reply:
Now protobuf in edge is used as "preinstalled" set of libs and headers (in the `install/` folder).
So the sources are useless here. It is not clear, what approach should I choose?
1. Create scripts to build appropriate libs and headers from sources?
2. Add compiled libs and keep shared headers?

Reply:
Customize build scripts and build from source for now. In general, we shall avoid putting big binaries into git. Especially, if they change frequently. For infrequent, can be fine though.